src/Migrations/20200210114133-create-node-past-deals.js

```diff
@@ -0,0 +1,52 @@
+var sequelize = require("sequelize");
+
+module.exports = {
+    up: function (queryInterface) {
+        return queryInterface.createTable('NodePastDeals', {
+            id: {
+                allowNull: false,
+                autoIncrement: true,
+                primaryKey: true,
+                type: sequelize.INTEGER
+            },
+            cid: {
+                type: sequelize.STRING
+            },
+            state: {
+                type: sequelize.INTEGER
+            },
+            size: {
+                type: sequelize.STRING
+            },
+            provider: {
+                type: sequelize.STRING
+            },
+            price: {
+                type: sequelize.STRING
```

Comment:
shouldn't here be BIGINT?

Reply:
Yes, my mistake. In the application it is a string, but in the database bigint.